packages/core/src/queries/connector.test.ts

```diff
@@ -27,13 +27,16 @@ describe('connector queries', () => {
 
   it('findAllConnectors', async () => {
     const rowData = { id: 'foo' };
-    const expectSql = sql`
-      select ${sql.join(Object.values(fields), sql`, `)}
-      from ${table}
+
+    /* eslint-disable sql/no-unsafe-query */
```

Comment:
It is not a good idea to disable lint rule(s) in such a vast code block.
maybe need to break this lint disable into smaller blocks if it is necessary.

Reply:
This lint rule SHOULD be always used in non-test code and sometimes needed in test code.

Reply:
Like this?

```ts
    /* eslint-disable sql/no-unsafe-query */
    const expectSql = `
      select "id", "enabled", "config", "created_at"
      from "connectors"
      order by "enabled" desc, "id" asc
    `;
    /* eslint-enabled sql/no-unsafe-query */
```